### ISSUE TITLE
wpscan brute force fails when server returns 302 redirect

### ISSUE THREAD

**Issue by jajarven** [wpscanteam/wpscan#1076]
When brute forcing and getting to correct username/password, but the server responds with 302 redirect, wpscan doesn't realize it got the right password: 
  Trying Username: admin Password: test                                                                         
  Incorrect login and/or password.                                                                             
  Trying Username: admin Password: admin                                                                       
  [!] ERROR: We received an unknown response for admin...                                                      
  [!]   Code: 302                                                                                              
    Body: 
  Brute Forcing 'admin' Time: 00:00:00 <==================================      > (6 / 7) 85.71%  ETA: 00:00:00

  +----+-------+------+----------+
  | Id | Login | Name | Password |
  +----+-------+------+----------+
  |    | admin |      |          |
  +----+-------+------+----------+

Server response starts with: HTTP/1.1 302 Found
wpscan version in use is 2.9.2

It would be nice to be able to handle other kind of response codes besides 200

**Comment by borneout** [comment]
i got the same issues like that, can someone help to resolve the issues ?

**Comment by ethicalhack3r** [comment]
Are you seeing this behaviour on a vanilla install of the latest version of WordPress? Or might there be some Plugins causing the behaviour?

**Comment by arall** [comment]
Similar issue here with vanilla install on the latest version:

```
[+] Enumerating plugins from passive detection ...
[+] No plugins found
[+] Starting the password brute forcer
  [!] ERROR: We received an unknown response for password...                                                                                                  
  Brute Forcing 'user' Time: 00:00:03 <=================================================================================    > (97 / 101) 96.03%  ETA: 00:00:00

  +----+-------+------+----------+
  | Id | Login | Name | Password |
  +----+-------+------+----------+
  |    | user  |      |          |
  +----+-------+------+----------+

[+] Finished: Mon Oct 30 06:57:04 2017
[+] Requests Done: 147
[+] Memory used: 17.012 MB
[+] Elapsed time: 00:00:07

```

**Comment by erwanlr** [comment]
This one should have been fixed in 3.5.0+

Feel free to open a new issue if you still have the issue using the latest 3.x version
